src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -357,6 +358,11 @@ private void initShuffleboard() {
 
 		angleSetpointEntry =
 				Shuffleboard.getTab("Launcher").add("Angle Setpoint", 0).withPosition(2, 2).getEntry();
+
+		Shuffleboard.getTab("Launcher")
+				.addDouble("setpoint + tol", () -> angleSetpoint - RPM_TOLERANCE);
```

Comment:
The wording doesn't match the code? (+ vs -)
Also, this should have a position (and size) specified to make sure this doesn't overlap with other entries.

Reply:
removed cuase i dont need it anymore